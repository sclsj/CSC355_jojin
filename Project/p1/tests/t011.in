record _identifiers_can_start_with_underscore_fields_on_one_line
{
  _a1 = "string a1"; _a2 = "string a2"; _a3 = "string a3";
}
