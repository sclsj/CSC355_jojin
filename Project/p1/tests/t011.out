parser.cpp::main() reading file tests/t011.in
parser.cpp::main() about to call yyparse()
type zero or more record declarations followed by ^D:

------------------------------------------------------
record _identifiers_can_start_with_underscore_fields_on_one_line
{

  _a1 = 'string a1' (string)
  _a2 = 'string a2' (string)
  _a3 = 'string a3' (string)

} 3 fields were declared.

1 record was declared.
------------------------------------------------------

parser.cpp::main() after call to yyparse()

no errors found by yyparse()

parser.cpp::main() end of program.
